config/initializers/pusher.rb

```diff
@@ -0,0 +1,4 @@
+require 'pusher'
+
+Pusher.url = "https://#{ENV["PUSHER_KEY"]}:#{ENV["PUSHER_SECRET"]}@api.pusherapp.com/apps/#{ENV["PUSHER_APP_ID"]}"
```

Comment:
Line is too long. [114/80]<br>Prefer single-quoted strings inside interpolations.